app/helpers/users_helper.rb

```diff
@@ -0,0 +1,27 @@
+module UsersHelper
+  def friendship_status_check(user_id)
+    friendship_status(user_id) if current_user != user_id
+  end
+
+  def friendship_status(user_id)
+    if current_user.friend?(user_id)
+      link_to 'Unfriend', friendship_path(user_id, method_name: 'delete'), method: :delete, class: 'friend_button'
+    elsif current_user.pend_friends.include?(user_id)
+      link_to 'Cancel', friendship_path(user_id, method_name: 'cancel'), method: :delete, class: 'friend_button'
+    elsif current_user.received_requests.include?(user_id)
+      # link_to 'Reject', friendship_path(user_id), method: :delete, class: 'friend_button'
+      # link_to 'Approve', friendship_path(user_id), method: :put, class: 'friend_button'
```

Comment:
- [ ] Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history